src/components/App/App.tsx

```diff
@@ -0,0 +1,16 @@
+import { H5PContentType } from "h5p-utils";
+import React from "react";
+import { Params } from "../../types/types";
+import { VocabularyDrill } from "../VocabularyDrill/VocabularyDrill";
+
+type AppProps = {
+  title: string;
+  contentId: string;
+  context: H5PContentType<Params>;
+};
+
+export const App: React.FC<AppProps> = ({ title, contentId, context }) => {
+  return (
+    <VocabularyDrill title={title} contentId={contentId} context={context} />
+  );
+};
```

Comment:
We _can_ keep the `App` component, or we could just use the `VocabularyDrill` component directly in `h5p-vocabulary-drill.tsx`

Reply:
Ooooo, then maybe just use `VocabularyDrill`